modules/codegen/src/main/scala/com/twilio/guardrail/generators/Java/JacksonGenerator.scala

```diff
@@ -103,7 +103,7 @@ object JacksonGenerator {
           case (value, termName, _) =>
             new EnumConstantDeclaration(
               new NodeList(),
-              new SimpleName(termName.getIdentifier.toSnakeCase.toUpperCase(Locale.US)),
```

Comment:
This is in "Add FormatEnumName term to ScalaTerms", but it doesn't appear to be doing that, what's going on here?

Reply:
Without this change it'll generate very wrong names with the addition of a (correct) `FormatEnumName` for Java.  In general I like to make each commit as correct as possible.

Reply:
Ah, due to the double snake case. Makes sense. Thanks!